packages/sdk/src/services/credential-service.ts

```diff
@@ -22,8 +21,21 @@ export interface GenerateCredentialExtendedDto extends Omit<GenerateCredentialDt
   issuer: string | { id: string; name?: string };
 }
 
+export interface CredentialDetail {
+  issuer: any | null;
```

Comment:
@vijayshiyani same here, `any` should not be used. It is either `string` or object with an `name` property `{name: string}`

Reply:
same as here  https://github.com/Meeco/js-sdk/pull/279/files#r1457138459